README.md

```diff
@@ -13,7 +13,7 @@
 
 A collection of simplified and community-driven man pages.
 
-Install it with `npm install -g tldr` or [try the web client](http://tldr.ostera.io).
+Install it with `npm install -g tldr`, `brew install tldr` (on OSX, using [Homebrew](http://brew.sh)), or [try the web client](http://tldr.ostera.io).
```

Comment:
My two cents: I would remove this line, as there are installation instructions in the Clients section already. If the Node client is the official one, this could be mentioned there too...